server/models/Profile.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose')
+
+const profileSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  availabiltity: {
+    type: Boolean,
+    default: false,
+  }
+}, { timestamps: true })
+
+module.exports = Profile = mongoose.model("profile", profileSchema);
```

Comment:
```suggestion
module.exports = Profile = mongoose.model("Profile", profileSchema);
```
Optional: Please use PascalCase for model names :)